bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/hbasesnapshots/ImportJobFromHbaseSnapshot.java

```diff
@@ -180,6 +180,19 @@ public interface ImportOptions extends ImportJob.ImportOptions {
     int getFilterLargeRowKeysThresholdBytes();
 
     void setFilterLargeRowKeysThresholdBytes(int value);
+
+    @Description(
+        "Specifies the path to the restored Snapshot files.")
+    String getRestorePath();
+
+    void setRestorePath(String value);
+
+    @Description("Deletes the restored snapshots.")
+    @Default.Boolean(false)
+    Boolean getDeleteRestoredSnapshots();
+
+    @SuppressWarnings("unused")
+    void setDeleteRestoredSnapshots(Boolean value);
```

Comment:
this shouldnt be a separate option - if the restore path is set, it should not be deleted. If its not set, it will be dynamically created in some random directory name so should be deleted because hunting down that directory is not a fun activity